.github/workflows/test-stories.yaml

```diff
@@ -117,6 +117,7 @@ jobs:
           token: ${{ github.token }}
           projectToken: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
           storybookBuildDir: "./docs/storybook-static"
+          storybookBaseDir: "./docs"
```

Comment:
So I think the storybook config _needs_ to be in a `.storybook/` directory, and I kinda like it haha

Reply:
I don't think it _needs_ to be :P But it is nicer anyway so I approve the change 👍🏻